src/views/related-assets-view/layout.tsx

```diff
@@ -1,76 +1,18 @@
-import React, { useEffect, useState } from "react";
+import React, { ReactNode } from "react";
 import { Link as RouterLink } from "react-router-dom";
 
-import { RelatedAssets } from "../../components/related-assets/related-assets";
-import { RelatedAsset, getAllRelatedAssets, organiseRelatedAssetsByType } from "./utils";
-
-import { Asset, ParentAsset } from "@mtfh/common/lib/api/asset/v1";
-import { Center, Link, Spinner } from "@mtfh/common/lib/components";
+import { Asset } from "@mtfh/common/lib/api/asset/v1";
+import { Link } from "@mtfh/common/lib/components";
 
 import "./styles.scss";
 
 interface RelatedAssetsLayoutProps {
   asset: Asset;
-  parentAssets: ParentAsset[];
-  childrenAssets: Asset[] | undefined;
-  loading: boolean;
+  children: ReactNode;
 }
 
-export const RelatedAssetsLayout = ({
-  asset,
-  parentAssets,
-  childrenAssets,
-  loading,
-}: RelatedAssetsLayoutProps): JSX.Element => {
-  const [relatedAssets, setRelatedAssets] = useState<RelatedAsset[]>([]);
-  const [relatedAssetsByType, setRelatedAssetsByType] = useState<any>(undefined);
-
-  useEffect(() => {
-    if (parentAssets?.length || childrenAssets) {
-      setRelatedAssets(getAllRelatedAssets(parentAssets, childrenAssets));
-    }
-  }, [parentAssets, childrenAssets]);
-
-  useEffect(() => {
-    if (relatedAssets.length) {
-      const assetsByType = organiseRelatedAssetsByType(relatedAssets);
-      setRelatedAssetsByType(assetsByType);
-    }
-  }, [relatedAssets]);
-
-  const assetHasRelatedAssets = (): boolean => {
-    return !!relatedAssets.length;
-  };
-
-  const renderRelatedAssets = () => {
-    if (loading) {
-      return (
-        <Center>
-          <Spinner />
-        </Center>
-      );
-    }
-
-    // If we have related assets and the relatedAssetsByType object we render RelatedAssets components for each Asset type
-    if (assetHasRelatedAssets() && relatedAssetsByType) {
-      return Object.keys(relatedAssetsByType).map((assetType) => {
-        return (
-          <section key={assetType}>
-            <RelatedAssets
-              assetType={assetType}
-              relatedAssets={relatedAssetsByType[assetType]}
-            />
-            <hr style={{ borderTop: "1px solid #e7eaec" }} />
-          </section>
-        );
-      });
-    }
-    return (
-      <p className="lbh-body-m" data-testid="no-related-assets-message">
-        There are no related assets for this property.
-      </p>
-    );
-  };
+export const RelatedAssetsLayout = (props: RelatedAssetsLayoutProps): JSX.Element => {
+  const { asset, children } = props;
```

Comment:
What's the component for? If it's just a header and a direct reference to a list of children, it's not necessary. It's just adding cognitive complexity for the sake of complexity. Remove it.

Reply:
The existing project structure contains a views directory, which contains a layout and view component. The difference between these are vague. However, my interpretation is that the view component acts as the page, handling query parameters, validation, authorization etc. Sometimes it loads data. The layout component holds the structure of the page.

The main change I made was to move the application logic away from the views directory, and into its own component, leaving this component fairly empty. 

Removing the layout component would go against the project's structure. I wouldn't agree that it's adding more complexity because it matches the structure of the other views.